package.json

```diff
@@ -39,5 +40,8 @@
 		"rollup": "^2.55.0",
 		"typescript": "^4.3.5"
 	},
-	"type": "module"
+	"type": "module",
+	"dependencies": {
+		"action-deploy-docs": "github:sveltejs/action-deploy-docs#main"
```

Comment:
should this be in `devDependencies` instead?

Reply:
yip